packages/web/docs/src/app/product-updates/(posts)/2025-05-20-advanced-breaking-changes/page.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Input Based Breaking Change Detection
+description:
+  Improvements to Hive's breaking change detection to make it easier and safer to migrate schemas.
+date: 2025-05-20
+authors: [jdolle]
+---
+
+## What’s New?
+
+- **Hive accounts for argument, input field, and variable values when checking for breaking
+  changes** – This improvement is fully backwards compatible and will automatically take effect when
+  upgrading your hive client package.
```

Comment:
I think it worth mention here the version that supports it? (`>=N`)

Reply:
Good suggestion. Will do.